static/app/components/onboarding/gettingStartedDoc/types.ts

```diff
@@ -54,6 +59,10 @@ export interface DocsParams<
     screenshot?: boolean;
   };
   newOrg?: boolean;
+  /**
+   * The page where the docs are being displayed
+   */
+  page?: DocsPageLocation;
```

Comment:
hmm, would choose something more generic than `page`. As e.g. a sidebar may be opened on different pages.
Maybe `docsLocation`?